draft-ietf-dnsop-svcb-httpssvc.md

```diff
@@ -1024,8 +1036,8 @@ be populated with the registrations below:
 
 | SvcParamKey | NAME        | Meaning                       | Reference       |
 | ----------- | ------      | ----------------------        | --------------- |
-| 0           | key0        | Reserved                      | (This document) |
-| 1           | alpn        | ALPN for alternative service  | (This document) |
+| 0           | proto       | Supported protocol            | (This document) |
+| 1           | no-proto    | Unsupported protocol          | (This document) |
```

Comment:
This could go either way, but since there are existing implementations with supported protocols at codepoint 1, wouldn't it make sense to put `no-proto` at 0 instead?